examples/embeddable_examples/public/app/presentation_container_example/components/presentation_container_example.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect, useMemo } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiSuperDatePicker,
+} from '@elastic/eui';
+import { useBatchedPublishingSubjects } from '@kbn/presentation-publishing';
+import { ReactEmbeddableRenderer } from '@kbn/embeddable-plugin/public';
+import { UiActionsStart } from '@kbn/ui-actions-plugin/public';
+import { getParentApi } from '../parent_api';
+import { AddButton } from './add_button';
+import { TopNav } from './top_nav';
+import { lastSavedStateSessionStorage } from '../session_storage/last_saved_state';
+import { unsavedChangesSessionStorage } from '../session_storage/unsaved_changes';
+
+export const PresentationContainerExample = ({ uiActions }: { uiActions: UiActionsStart }) => {
+  const { cleanUp, componentApi, parentApi } = useMemo(() => {
+    return getParentApi();
+  }, []);
+
+  useEffect(() => {
+    cleanUp();
+  }, [cleanUp]);
+
+  const [dataLoading, panels, timeRange] = useBatchedPublishingSubjects(
+    parentApi.dataLoading,
+    componentApi.panels$,
+    parentApi.timeRange$
+  );
+
+  return (
+    <div>
+      <EuiCallOut title="Presentation Container interfaces">
+        <p>
+          At times, you will need to render many embeddables and allow users to add, remove, and
+          re-arrange embeddables. Use the <strong>PresentationContainer</strong> and{' '}
+          <strong>CanAddNewPanel</strong> interfaces for this functionallity.
+        </p>
+        <p>
+          Each embeddable manages its own state. The page is only responsible for persisting and
+          providing the last persisted state to the embeddable. Implement{' '}
+          <strong>HasSerializedChildState</strong> interface to provide an embeddable with last
+          persisted state. Implement <strong>HasRuntimeChildState</strong> interface to provide an
+          embeddable with previous sessions&apos;s unsaved changes.
```

Comment:
```suggestion
          embeddable with previous session&apos;s unsaved changes.
```

`sessions's` -> `session's`